src/main/webapp/gameFunctions.js

```diff
@@ -5,3 +5,39 @@ function getUserQuizStatus() {
         }
     });
 }
+
+/** 
+* fetches users of the game from the server 
+* and adds score, id and name to the page
+*/
+function loadUsers() {
+    fetch('/populate-leaderboard').then(response => response.json()).then((users) => {
+        const userListElement = document.getElementById('leader-board');
+        for(i = 0; i < users.length; i++){
+            userListElement.appendChild(createUserElement(users[i]));
+        }
+    });
+}
+ 
+/** Creates a list item that contains the user */
+function createUserElement(user){
+  
+  // create list elements
+  const userElement = document.createElement('li');
+  const userIDElement = document.createElement('div');
```

Comment:
Do you need line 27 calling the ID or should we just show the username?

Reply:
I figured we'd have both because 2 users can have the same name